src/main/webapp/js/issues-detail.js

```diff
@@ -178,72 +105,18 @@
+        const carousel = $('#' + carouselId);
         carousel.on('slid.bs.carousel', function (e) {
             localStorage.setItem(carouselId, e.to);
-            var chart = $(e.relatedTarget).find('>:first-child').data('chart');
-            chart.resize();
+            const chart = $(e.relatedTarget).find('>:first-child')[0].echart;
+            if (chart) {
+                chart.resize();
+            }
         });
-        var activeCarousel = localStorage.getItem(carouselId);
+        const activeCarousel = localStorage.getItem(carouselId);
         if (activeCarousel) {
             carousel.carousel(parseInt(activeCarousel));
         }
     }
 
-    /**
-     * Initializes the specified table.
-     *
-     * @param {String} id - the ID of the table
-     */
-    function showTable (id) {
-        var table = $(id);
-        if (table.length) {
-            // Create a data table instance for the issues table.
-            var dataTable = table.DataTable({
-                language: {
-                    emptyTable: 'Loading - please wait ...'
-                },
-                deferRender: true,
-                pagingType: 'numbers', // Page number button only
-                order: [[1, 'asc']],
-                columnDefs: [{
-                    targets: 0, // First column contains details button
-                    orderable: false
-                }],
-                columns: JSON.parse(table.attr('data-columns-definition'))
-            });
-
-            // Add event listener for opening and closing details
-            table.on('click', 'div.details-control', function () {
-                var tr = $(this).parents('tr');
-                var row = dataTable.row(tr);
-
-                if (row.child.isShown()) {
-                    // This row is already open - close it
-                    row.child.hide();
-                    tr.removeClass('shown');
-                }
-                else {
-                    // Open this row
-                    row.child($(this).data('description')).show();
-                    tr.addClass('shown');
-                }
-            });
-
-            // Content is loaded on demand: if the active tab shows the table, then content is loaded using Ajax
-            var tabToggleLink = $('a[data-toggle="tab"]');
-            tabToggleLink.on('show.bs.tab', function (e) {
-                var activeTab = $(e.target).attr('href');
-                if (activeTab === (id + 'Content') && dataTable.data().length === 0) {
-                    view.getTableModel(id, function (t) {
-                        (function ($) {
-                            var table = $(id).DataTable();
-                            var model = JSON.parse(t.responseObject());
-                            table.rows.add(model).draw();
-                        })(jQuery);
-                    });
-                }
-            });
-        }
-    }
-})(jQuery);
+})(jQuery3);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must not be padded by blank lines.](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=4663559)